pkg/framework/coremodel/staticregistry/provide.go

```diff
@@ -3,15 +3,28 @@ package staticregistry
 import (
 	"github.com/grafana/grafana/pkg/coremodel/dashboard"
 	"github.com/grafana/grafana/pkg/framework/coremodel"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
 )
 
 // ProvideRegistry provides a simple static Registry for coremodels.
 // Coremodels have to be manually added.
 // TODO dynamism
 func ProvideRegistry(
+	feat featuremgmt.FeatureManager,
 	dashboard *dashboard.Coremodel,
 ) (*coremodel.Registry, error) {
-	return coremodel.NewRegistry(
+	cmlist := []coremodel.Interface{
 		dashboard,
-	)
+	}
+
+	// return coremodel.NewRegistry(cmlist...)
+	// Uncomment above line and everything delete below, as well as unfeatured.go,
+	// once coremodels are no longer feature flagged
+	var newfunc registryProvider
+	newfunc = coremodel.NewRegistry
+	if feat.IsEnabled(featuremgmt.FlagCoremodelValidation) {
+		newfunc = provideStub
+	}
```

Comment:
This is kind of a lie. We're actually stubbing out the _whole_ `coremodel.Interface` object, which means swapping in a whole faked-out Thema lineage. This is sufficient for the purposes of this PR, where we're only doing the simplest possible thing - `cm.CurrentSchema().Validate()`. But it gets complicated as we want to do more.

(have to run right now, but will elaborate further later today)